wwwroot/inc/functions.php

```diff
@@ -38,6 +38,7 @@
 define ('RE_L2_IFCFG', '/^[0-9A-F]{2}(:[0-9A-F]{2}){5}$/'); // most ifconfigs
 define ('RE_L2_IFCFG_SUNOS', '/^[0-9A-F]{1,2}(:[0-9A-F]{1,2}){5}$/'); // SunOS ifconfig
 define ('RE_L2_CISCO', '/^[0-9A-F]{4}(\.[0-9A-F]{4}){2}$/');
+define ('RE_L2_HP', '/^[0-9A-Fa-z]{6}(-[0-9A-Fa-z]{6}){1}$/');
```

Comment:
0-9A-Fa-f of course

Reply:
even "0-9A-F" (see similar regexps, there is a `strtoupper()` somewhere)

Also `{1}` is a no-op, so the regexp is as simple as: `/^[0-9A-F]{6}-[0-9A-F]{6}$/`